t/001-wildcard-router.t

```diff
@@ -0,0 +1,113 @@
+use Test::Nginx::Socket::Lua 'no_plan';
+use Cwd qw(cwd);
+
+my $pwd = cwd();
+my $nginx = $ENV{TEST_NGINX_ROOT_PATH} || "$pwd";
+
+$ENV{TEST_NGINX_LUA_PATH} = "$nginx/src/?.lua;$nginx/src/?/init.lua;;";
+$ENV{TEST_NGINX_UPSTREAM_CONFIG} = "$nginx/nginx/upstream_wildcard.conf";
+$ENV{TEST_NGINX_DEFAULT_CONFIG} = "$nginx/nginx/default_wildcard.conf";
+$ENV{TEST_NGINX_SERVICE_CONFIG} = "$nginx/nginx/service_wildcard.conf";
+
+require("t/dns.pl");
+
+log_level('debug');
+repeat_each(2);
+no_root_location();
+run_tests();
+
+env_to_nginx(
+  'RESOLVER',
+  'API_HOST',
+  'ACCESS_TOKEN',
+  'APICAST_PORT'
+);
+
+__DATA__
+
+# === TEST 1: request though the wildcard router
+
+# --- main_config
```

Comment:
This test is commented. So the normal flow when the route matches apicast-staging|production is not tested.